src/main/java/journeymap/client/ui/waypoint/WaypointManager.java

```diff
@@ -114,34 +178,7 @@ public void initGui()
             }
             buttonList.add(buttonToggleAll);
 
-            // Bottom buttons
-            if (buttonDimensions == null)
-            {
-                buttonDimensions = new DimensionsButton();
-            }
-
-            if (buttonAdd == null)
-            {
-                buttonAdd = new Button(Constants.getString("jm.waypoint.new"));
-                buttonAdd.fitWidth(getFontRenderer());
-                buttonAdd.setWidth(buttonAdd.getWidth() * 2);
-//                String tooltip = Constants.getString("jm.waypoint.new.tooltip");
-//                buttonAdd.setTooltip(tooltip);
-            }
-
-            if (buttonOptions == null)
-            {
-                buttonOptions = new Button(Constants.getString("jm.common.options_button"));
-                buttonOptions.fitWidth(getFontRenderer());
-//                String tooltip = Constants.getString("jm.waypoint.help.tooltip");
-//                buttonOptions.setTooltip(tooltip);
-            }
-
-            buttonClose = new Button(Constants.getString("jm.common.close"));
-
-            bottomButtons = new ButtonList(buttonOptions, buttonAdd, buttonDimensions, buttonClose);
-            buttonList.addAll(bottomButtons);
-
+            init.run();
```

Comment:
why are these runnable? I am not a fan of that.

Reply:
this is a basic method for initializing 2 different Gui.
One Gui is the main `initGui`, and another method `initConfirmGui` it's a method for initializing confirming buttons to remove all waypoints.

I need to run this code because I don't want to repeat the same code.
It's allowed me to write that code.

```Java
    @Override
    public void initGui()
    {
        initBasicGui(() -> {
            // here some code
            // for instance I init buttons which I need
        });
    }

    private void initConfirmGui()
    {
        initBasicGui(() -> {
            // here another code
        });
    }
```

maybe it isn't correct in Java but I'm a .NET developer, so it looks correct to me

Reply:
I get what it's doing and why. 
Seems like a .net thing haha. it works, it's fine
Thanks for your submission, I will merge it an publish a new release